Sources/Modules/FilesModule.swift

```diff
@@ -314,6 +314,7 @@ public class FilesModule {
     ///   - name: The name of the file. Box supports file names of 255 characters or
     ///     less. Names containing non-printable ASCII characters, "/" or "\", names with trailing
     ///     spaces, and the special names “.” and “..” are also not allowed.
+    ///   - ifMatch: The `etag` of the file version. Ensures this item hasn't recently changed before making changes.
```

Comment:
Just riffing here, to make the usage clear. Not insisting on a change.
```suggestion
    ///   - ifMatch: The `etag` of the old file version. Ensures that the file hasn't been updated since getting the etag, which could indicate conflicting edits.
```